### ISSUE TITLE
Does new package name require changes to associated go libraries

### ISSUE THREAD

**Issue by ukclivecox** [apache/arrow-go#182]
### Describe the bug, including details regarding any error messages, version, and platform.

See https://github.com/apache/arrow-adbc/issues/2290

### Component(s)

Integration

**Comment by zeroshade** [comment]
Yes, the new package name does require changes to the associated go libraries.

**Comment by zeroshade** [comment]
Gonna close this, feel free to reopen if necessary
